<launch>
    <!-- Run joint trajectory action server node (required for MoveIt!) -->
    <node pkg="baxter_interface" type="joint_trajectory_action_server.py" name="trajectory_node" >
    </node>
    <!-- Run launch file that runs MoveIt!. Remap the joint state topic -->
    <include file="$(find baxter_moveit_config)/launch/baxter_grippers.launch"/>
      <remap from="joint_states" to="robot/joint_states"/>
</launch>
